CASMINST-55245 - change references to csm versions again in release 1… (#2889)
Cray-HPE/docs-csm: install/index.md

# Install CSM

> Note: If you are beginning an install with CSM 1.2.0, please download and proceed to install the latest released CSM 1.2.x version, instead. This will save you time performing a patch update later.

Reviewer: ```suggestion
Note: If installing CSM 1.2.0, then in order to avoid performing a patch update later, instead install the latest released CSM 1.2.x version.
```
